docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -24,7 +24,7 @@ Defaults to the value set below.
 ```xml
 <configuration>
   <svcsAnnotationsFile>
-    ${project.build.directory}/generated-test-sources/test-annotations/resources/annotations
+    ${project.build.directory}/generated-test-sources/test-annotations/resources/annotations.yml
   </svcsAnnotationsFile>
 </configuration>
```

Comment:
Seem to be missing the rest of the docs on how to configure using `<configuration>`.